nazgul/api.py

```diff
@@ -2,8 +2,10 @@
 
 try:
     from nazgul.mysql_model import MySQLModel
+    import nazgul.xmlrenderer as xmlrenderer
 except:
     from mysql_model import MySQLModel
+    import xmlrenderer
```

Comment:
I think the try statement should be removed and it should just be `from . import xmlrenderer`, for example.

Reply:
I did this because I could not find a way to run both flask on docker and pytest with and import statement that works for both

Reply:
We should do this: http://flask.pocoo.org/docs/1.0/tutorial/install/ so the imports always work correctly.

Reply:
I got it to work without doing the making the flask app installable.